docs/configuration.md

```diff
@@ -1,10 +1,19 @@
-# Configuration (WIP)
+# Configuration
 
 ## Docker Basic Configuration
+
 ```
-NEKO_PASSWORD=neko      // Password
-NEKO_ADMIN=neko         // Admin Password
+NEKO_ADMIN=admin        // Admin Password
 NEKO_BIND=0.0.0.0:8080  // Bind
-NEKO_KEY=               // (SSL) Key, needed for clipboard sync
 NEKO_CERT=              // (SSL) Cert, needed for clipboard sync
-```
\ No newline at end of file
+NEKO_KEY=               // (SSL) Key, needed for clipboard sync
+NEKO_PASSWORD=neko      // Password
+```
+
+## Environment Variables
+
+* **NEKO_ADMIN**:  the administrative user password. Any user authenticating with this will be given admin privileges.
```

Comment:
everything looks good, but recently changed this to `NEKO_PASSWORD_ADMIN`

Reply:
Done, updated doc to reflect that change.